src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -700,8 +698,21 @@ private void processNextFile() {
     }
   }
 
+  private void postUpload() {
```

Comment:
Refactored some code for reuse.  This same code is run if the upload resulted in a null fileHandleId, or if we successfully created the fileHandle and wrapped it with a file entity.